src/taskgraph/util/taskcluster.py

```diff
@@ -367,3 +368,22 @@ def list_task_group_incomplete_tasks(task_group_id):
             params = {"continuationToken": resp.get("continuationToken")}
         else:
             break
+
+
+def get_recursive_dependencies(
+    task_ids: List[str], add_given_tasks: bool = False, use_proxy: bool = False
+) -> Dict[str, str]:
+    """Returns a list of all the recursive dependencies of the task_ids provided as
+    a dictionary of a dictionary of label -> taskId."""
```

Comment:
Couple typos here:

* s/Returns a list of all/Returns all
* s/a dictionary of a dictionary/a dictionary

Would also be nice to explain the args in a format like `def state_task` has.